tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java

```diff
@@ -105,13 +106,13 @@ public void singleMasterJournalCrashIntegration() throws Exception {
   @Test
   public void multiMasterJournalStopIntegration() throws Exception {
     MultiMasterLocalAlluxioCluster cluster = setupMultiMasterCluster();
+    CommonUtils.sleepMs(TEST_TIME_MS);
```

Comment:
does `setupMultiMasterCluster` return before the cluster is ready? I wonder if we could use more fine-grained waiting for cluster startup